File: knowledgebase-sdk/src/main/java/ru/usedesk/knowledgebase_sdk/external/entity/UsedeskSection.java
package ru.usedesk.knowledgebase_sdk.external.entity;

import com.google.gson.annotations.SerializedName;

public class UsedeskSection {

    private long id;
    private String title;
    @SerializedName("public")
    private int access;
    private int order;
    private String image;
    private UsedeskCategory[] categories;

    public long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public int getAccess() {
        return access;
    }

    public int getOrder() {
        return order;
    }

    public String getImage() {
        return image;
    }

    public UsedeskCategory[] getCategories() {
        return categories;
    }
}

File: knowledgebase-sdk/src/main/java/ru/usedesk/knowledgebase_sdk/external/entity/UsedeskSection.kt
package ru.usedesk.knowledgebase_sdk.external.entity

import com.google.gson.annotations.SerializedName

class UsedeskSection {
    val id: Long = 0
    val title: String? = null

    @SerializedName("public")
    val access = 0
    val order = 0
    val image: String? = null
    var categories: Array<UsedeskCategory>? = null
}